rb/lib/selenium/webdriver/remote/commands.rb

```diff
@@ -25,191 +25,194 @@ class Bridge
+            get_app_cache_status: [:get, 'session/:session_id/application_cache/status'.freeze],
+            clear_app_cache: [:delete, 'session/:session_id/application_cache/clear'.freeze],
+
+            get_network_connection: [:get, 'session/:session_id/network_connection'.freeze],
+            set_network_connection: [:post, 'session/:session_id/network_connection'.freeze],
+
+            get_local_storage_item: [:get, 'session/:session_id/local_storage/key/:key'.freeze],
+            remove_local_storage_item: [:delete, 'session/:session_id/local_storage/key/:key'.freeze],
+            get_local_storage_keys: [:get, 'session/:session_id/local_storage'.freeze],
+            set_local_storage_item: [:post, 'session/:session_id/local_storage'.freeze],
+            clear_local_storage: [:delete, 'session/:session_id/local_storage'.freeze],
+            get_local_storage_size: [:get, 'session/:session_id/local_storage/size'.freeze],
+
+            get_session_storage_item: [:get, 'session/:session_id/session_storage/key/:key'.freeze],
+            remove_session_storage_item: [:delete, 'session/:session_id/session_storage/key/:key'.freeze],
+            get_session_storage_keys: [:get, 'session/:session_id/session_storage'.freeze],
+            set_session_storage_item: [:post, 'session/:session_id/session_storage'.freeze],
+            clear_session_storage: [:delete, 'session/:session_id/session_storage'.freeze],
+            get_session_storage_size: [:get, 'session/:session_id/session_storage/size'.freeze],
+
+            #
+            # ime
+            #
+
+            ime_get_available_engines: [:get, 'session/:session_id/ime/available_engines'.freeze],
+            ime_get_active_engine: [:get, 'session/:session_id/ime/active_engine'.freeze],
+            ime_is_activated: [:get, 'session/:session_id/ime/activated'.freeze],
+            ime_deactivate: [:post, 'session/:session_id/ime/deactivate'.freeze],
+            ime_activate_engine: [:post, 'session/:session_id/ime/activate'.freeze],
+
+            #
+            # touch
+            #
+
+            touch_single_tap: [:post, 'session/:session_id/touch/click'.freeze],
+            touch_double_tap: [:post, 'session/:session_id/touch/doubleclick'.freeze],
+            touch_long_press: [:post, 'session/:session_id/touch/longclick'.freeze],
+            touch_down: [:post, 'session/:session_id/touch/down'.freeze],
+            touch_up: [:post, 'session/:session_id/touch/up'.freeze],
+            touch_move: [:post, 'session/:session_id/touch/move'.freeze],
+            touch_scroll: [:post, 'session/:session_id/touch/scroll'.freeze],
+            touch_flick: [:post, 'session/:session_id/touch/flick'.freeze],
+
+            #
+            # logs
+            #
+
+            get_available_log_types: [:get, 'session/:session_id/log/types'.freeze],
+            get_log: [:post, 'session/:session_id/log'.freeze]
+        }
```

Comment:
Without my running specs for this yet, can we freeze this constant now, or does Edge still make this difficult?